vip-cache-manager/vip-cache-manager.php

```diff
@@ -285,7 +291,7 @@ public function queue_post_purge( $post_id ) {
 		if ( $this->site_cache_purged )
 			return false;
 
-		if ( defined( 'WP_IMPORTING' ) ) {
+		if ( ( defined( 'WP_IMPORTING' ) && true === WP_IMPORTING ) ) {
```

Comment:
```suggestion
		if ( defined( 'WP_IMPORTING' ) && true === WP_IMPORTING ) {
```

Think we have an extra set of parenthesis here.